### DISCUSSION TITLE
How to determine if the workflow failed

### DISCUSSION THREAD

**Discussion by balchua** [cschleiden/go-workflows#323] [Q&A] [answered]
I have a case where i need to show that the workflow failed, but the state only shows completed.

How do one get failed workflow?

**Comment by cschleiden** [comment, marked as answer]
You can return an error and get the result of the workflow.

**Reply by balchua** [reply]
Thanks.  Sorry newbie question, do you have an example somewhere in the code that can allow one to get the result of the workflow?

So what im doing is, the workflow application exposes an endpoint that can start a workflow and return.  It also exposes an endpoint to get the status of the workflow by passing it the instanceid and executionId.

For the caller to know the status of the workflow, it will just hit the status endpoint.

**Reply by balchua** [reply]
Was looking at the client code, is it `GetWorkflowResult` to get the outcome of the workflow?

**Reply by cschleiden** [reply]
Yep!
